src/extension/views/json/json.js

```diff
@@ -352,7 +353,7 @@ export class JSONView extends LitElement {
         valueContainer.classList.add('number');
         valueContainer.textContent = value;
       }
-    } else if (/\/^\/[a-z0-9]+$\/i/.test(value) || value.startsWith('http')) {
+    } else if (/^\/[a-z0-9]+/i.test(value) || value.startsWith('http')) {
```

Comment:
Fixes a regression introduced back in #432 which prevented links and linked images from being rendered as expected.